arm-web/2016-08-01/swagger/WebApps.json

```diff
@@ -14309,47 +14553,36 @@
     "VnetInfo": {
       "description": "Virtual Network information contract.",
       "type": "object",
-      "allOf": [
-        {
-          "$ref": "#/definitions/Resource"
```

Comment:
fine, check properties can be shared as described above [AppServiceCertificateResource and AppServiceCertificate]